scripts/envhook.py

```diff
@@ -41,6 +41,16 @@ def main(argv):
         else:
             raise
     if options.action == 'install':
+        try:
+            import sitecustomize
+        except ImportError:
+            pass
+        else:
+            if os.path.realpath(sitecustomize.__file__) == os.path.abspath(__file__):
```

Comment:
`os.path.abspath(__file__)` is alrady available as `dst`. And I don't think we should mix realpath and abspath, we should use either one.

Also, not the [existing logic](https://github.com/DataBiosphere/azul/pull/1158/files#diff-446a633c96640bbd37d927917bf990ceL40) for checking. We should probably move this section up so it is run both when we install and remove. I think it should precede the code in line 20.

Reply:
`os.path.abspath(__file__)` is not already available as dst because it is turned into a relative path in the next line. I will change it so that a separate variable is used for the absolute and relative paths and the code isn't duplicated.

realpath and abspath do different things and both are necessary. realpath here resolves the link in the path to the `sitecustomize.__file__` and abspath gives the absolute path to the script that's running. Using realpath in both places would technically work, but it seems to me to be more explicit to use abspath for the current file.

Reply:
I see your last point, the code should be moved. I think it should be moved after line 30 so that you are warned about not having a virtualenv active before you are warned about the already installed sitecustomize.py. Feel free to message back on Slack for faster back and forth here.

Reply:
> os.path.abspath(__file__) is not already available as dst because it is turned into a relative path in the next line.

I missed that.

> I think it should be moved after line 30

Yes, that's better.

> realpath and abspath do different things

Realpath calls abspath on Darwin and Posix. I therefore think of one of them to be a superset of the other and it would be misleading to treat them as doing "different things".

> Using realpath in both places would technically work, but it seems to me to be more explicit to use abspath for the current file.

See my most recent review comment